### DISCUSSION TITLE
Getting "mounts denied" error when running docker

### DISCUSSION THREAD

**Discussion by Snorre83** [ptrumpis/snap-camera-server#51] [Q&A] [answered]
getting this message 👍

Attaching to snap-camera-server-311-adminer-1, snap-camera-server-311-mysql-1, snap-camera-server-311-webapp-1, snap-camera-server-311-webproxy-1
Error response from daemon: Mounts denied: 
The path /applications/snap-camera-server-3.1.1/mysql is not shared from the host and is not known to Docker.
You can configure shared paths from Docker -> Preferences... -> Resources -> File Sharing.

Anyone know a fix?

<img width="695" alt="Skjermbilde 2023-08-08 kl  14 56 24" src="https://github.com/ptrumpis/snap-camera-server/assets/141740772/94ba954e-bf29-4b83-b5cb-0cc6c30607eb">

**Comment by ptrumpis** [comment, marked as answer]
https://www.youtube.com/watch?v=b2ILHJaD1T4&t=430s

**Reply by Snorre83** [reply]
Thank you so much!! I just didn't watch the vid far enough! Amazing work!!

**Reply by Snorre83** [reply]
@ptrumpis Ok sorry, here we go again. I followed the instructions, but the same message keeps returning. Any clue?

**Reply by ptrumpis** [reply]
Are you sure you are getting the same error message? Or has anything changed?
Did you save the changes in your preferences?

**Reply by Snorre83** [reply]
Solved! I'm not 100% sure, but I think restarting Terminal was the trick.
